packages/gravity-ui-web/src/sass/04-objects/_sticky-footer.scss

```diff
@@ -0,0 +1,17 @@
+.grav-o-sticky-footer {
+  // By default, <body> has top and bottom padding, so we need to subtract that
+  // from the sticky footer container's min height.
+  @include grav-sticky-footer( calc(100vh - #{2 * $grav-sp-page-content-inset}) );
+
+  // Special case:
+  // When within a full-bleed body, that vertical padding is gone, so we
+  // need to adjust our min-height accordingly:
+  .grav-o-full-bleed & {
+    min-height: 100vh;
+  }
+}
+
+.grav-o-undo-sticky-footer {
+  display: block;
+  min-height: none;
+}
```

Comment:
Although it's not just just one property that's being overriden I'd say this may be better suited as a utility class - perhaps using a basic mixin (could be scaled up for other uses in the future). It does really lend itself more to a modifier (since it does one job and also only has one use case). A modifier with no block selector though is a bit awkward maybe? 🤔

Reply:
@dw-buildit Just had a chat about it and decided to go with a modifier. While it is an "odd" one in that you'd probably never use it together with the corresponding "block" class, it does feel like the most logical home for this. Hopefully, its also where developers might think to look for a way to undo the sticky footer behaviour.